examples/connector/python/build.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+# Creates virtual environment to install the packages and to run the connector.
+python3 -m venv connector_run
```

Comment:
We need to add `connector_run` directory to `.gitignore`

Reply:
Yes, I added it. I think while rebasing gitignore file got bit messed up due to IDE issue. I'll update it.